src/parser/codeParser.ts

```diff
@@ -5,11 +5,12 @@ const testTokens = ["suite", "describe", "context", "it", "specify", "test"];
 function codeParser(sourceCode) {
   const parserOptions: ParserOptions = {
     plugins: [
-      "jsx",
       "typescript",
-      "objectRestSpread",
-      "optionalChaining",
-      "nullishCoalescingOperator"
+      ["decorators", { decoratorsBeforeExport: false }],
+      "classPrivateMethods",
+      "classPrivateProperties",
+      "topLevelAwait",
+      "jsx"
```

Comment:
🤔 Are these necessary for any reason?

Reply:
I added additional - not yet default babel features but taht are available in v8/node today, that currently cause the plugin to fail